Taskfile.yml

```
@@ -217,8 +217,44 @@ tasks:
         -target-url=mongodb://127.0.0.1:47017/
         -target-backend=mongodb
 
+  bench-pg:
+    desc: "Run benchmarks for `pg` handler"
+    dir: integration
+    cmds:
+      - >
+        go test -bench='Benchmark.*' -run=^#
+        -benchtime={{.BENCHTIME}}
+        -target-backend=ferretdb-pg
+        -target-tls
+        -postgresql-url=postgres://username@127.0.0.1:5432/ferretdb
```

```suggestion
        -postgresql-url=postgres://username@127.0.0.1:5432/ferretdb
        | tee new-pg.txt
```

To actually generate that file